spirv_hlsl.cpp

```diff
@@ -3152,234 +3322,263 @@ void CompilerHLSL::emit_hlsl_entry_point()
 	bool legacy = hlsl_options.shader_model <= 30;
 
 	// Copy builtins from entry point arguments to globals.
-	active_input_builtins.for_each_bit([&](uint32_t i) {
```

Comment:
At a glance this also looks like a weird code reflow. Is there any functional change hidden in this diff?

Reply:
Just happened to be a function I touched, so clangformat was applied. I've reverted.